test/scenarios/istio/istio_test.go

```diff
@@ -34,11 +36,21 @@ func TestMain(m *testing.M) {
 	nsWithoutIstio := *namespace.New(operator.DefaultNamespace)
 	testEnv.BeforeEachTest(istio.AssertIstioNamespace())
 	testEnv.BeforeEachTest(istio.AssertIstiodDeployment())
-	testEnv.Setup(
-		helpers.SetScheme,
-		namespace.CreateForEnv(nsWithIstio),
-		operator.InstallViaMake(true),
-	)
+
+	if scenarios.InstallViaHelm() {
```

Comment:
I prefer to do it in more clear way:
- `make deploy` under the hood also installs the operator using Helm, but we use a local Helm chart.
https://github.com/dynatrace/dynatrace-operator/blob/87e0cdb07199b8069ba79fa74050c300588badc7/hack/make/deploy/deploy.mk#L12-L24
- so the only new we need to handle is install via helm but use custom params and OCI chart, so